include/ofxColourSpace.h

```diff
@@ -117,6 +117,15 @@ extern "C" {
 
 #define kOfxImageClipPropColourSpace "OfxImageClipPropColourSpace"
 
+/** @brief The approximate gamma value
+
+- Type - double X 1
+- Default: 1.0  - linear
+- Valid Values - can be any positive value, usual suspects are 2.2, 2.4, 2.6
+
+	This property is slightly orthogonal to the other ones in this property set. The main purpose is for a plug-in to internally linearize and delinearize non-destructively in a less arbitrary way.
```

Comment:
This needs a lot more clarification. Is this the gamma that the plug-in should expect its input to be encoded with? Is it valid on input and output? During what calls is it valid?
In general I don't think there is enough info in this header to effectively implement this suite on either plugin or host side yet. Can some props be specified and others not? What's the expected calling/checking sequence on both sides? Maybe add a couple of worked examples or a table in the comments, so those will get into the doc.

Reply:
Added some clarifications. Note this is exactly like we do in AE when user switches from project linearize versus color managed.  It does not imply any processing from host, it's just data for plugins.